### DISCUSSION TITLE
2024-09-03_화_잡담

### DISCUSSION THREAD

**Discussion by tttghost** [tttghost/wheelmaker#49] [General]
- 여름휴가 잘 다녀옴 (양양, 봉화)
- 몇번의 야근
- 잦은 술약속
- 아예 이번주는 릴렉스하게 보내보자 라고 중간지점부터 마음먹게 되었습니다.
- 차주부터 다시 스프린트하게

**Comment by jongfeel** [comment]
그러면 이번에는 진행한 내용이 없어 보이는데
맞다면 오늘 온라인 미팅을 미루는게 좋을까요?

**Reply by jisu-VIRNECT** [reply]
간단한 근황이야기는 어떠씰까요..?

**Reply by jongfeel** [reply]
네 그러면 30분 정도 얘기해 보면 좋겠습니다.

**Comment by tttghost** [comment]
네 업무상 궁금한것들도 좀 생기고 방향성도 궁금한게 생겻습니다! 미리몇가지 적어놓겠습니다!

iOS용 Outlook<https://aka.ms/o0ukef> 다운로드
________________________________
보낸 사람: Kim Jong Feel ***@***.***>
보낸 날짜: Wednesday, September 4, 2024 9:25:27 AM
받는 사람: tttghost/wheelmaker ***@***.***>
참조: 김지수 ***@***.***>; Author ***@***.***>
제목: Re: [tttghost/wheelmaker] 2024-09-03_화_잡담 (Discussion #49)


네 그러면 30분 정도 얘기해 보면 좋겠습니다.

**Comment by jisu-VIRNECT** [comment]
- UnityWebRequest 모듈
- 백단 만드는 작업도 병행
- 총 기간 설정
- UIManager의 정의
